src/Microsoft.MobileBlazorBindings.WebView/BlazorHybridHost.cs

```diff
@@ -5,14 +5,23 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using System.Reflection;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.MobileBlazorBindings.WebView.Android")]
```

Comment:
Icky. Would it be OK to just make the file provider collection public? Any harm in that?